topics/proteomics/tutorials/metaquantome-data-creation/tutorial.md

```diff
@@ -0,0 +1,656 @@
+
+
+> ### {% icon hands_on %} Hands-on: Extracting biological processes for metaQuantome
+>
+> 1. {% tool [Filter](Filter1) %} with the following parameters:
+>    - {% icon param-file %} *"Filter"*: `Unipept_Function` (output of **Query Tabular** {% icon tool %})
+>    - *"With following condition"*: `c5=='biological process'`
+>    - *"Number of header lines to skip"*: `1`
+>
+> 2. Rename file as go_bp
+{: .hands_on}
+
+
+![Biological Processes](../../images/biologicalprocess.png){: width="60%"}
+
+## *Filter* - Cellular components
+
+This step is used to filter out the GO terms with cellular components and the corresponding number of peptides associated with these terms.
+
+> ### {% icon hands_on %} Hands-on: Extracting cellular component for metaQuantome
+>
+> 1. {% tool [Filter](Filter1) %} with the following parameters:
+>    - {% icon param-file %} *"Filter"*: `Unipept_Function` (output of **Query Tabular** {% icon tool %})
+>    - *"With following condition"*: `c5=='cellular component'`
+>    - *"Number of header lines to skip"*: `1`
+>
+> 2. Rename file as go_cc
+{: .hands_on}
+
+
+![Cellular-Component](../../images/cellularcomponent.png){: width="60%"}
+
+## *Filter* - Molecular Function
+
+This step is used to filter out the GO terms with molecular function and the corresponding number of peptides associated with these terms.
+
+> ### {% icon hands_on %} Hands-on: Extracting molecular function for metaQuantome
+>
+> 1. {% tool [Filter](Filter1) %} with the following parameters:
+>    - {% icon param-file %} *"Filter"*: `Unipept_Function` (output of **Query Tabular** {% icon tool %})
+>    - *"With following condition"*: `c5=='molecular function'`
+>    - *"Number of header lines to skip"*: `1`
+> 2. Rename file as go_mf
+{: .hands_on}
+
+
+![Molecular-Function](../../images/molecularfunction.png){: width="60%"}
+
+# **Conclusion**
+{:.no_toc}
+
+This completes the walkthrough of the metaQuantome data creation workflow .This tutorial is a guide to have datasets that are metaQuantome ready/compatible and can be used for metaproteomics research. We have incorporated only two conditions in this workflow but users can use as many as they want. Researchers can use this workflow with their data also, please make sure the tool parameters and the workflow will be needed to be modified accordingly.
+
+Please look at the following tutorials in this Metaproteomics series:
+
+[Metaproteomics]({% link topics/proteomics/tutorials/metaproteomics/tutorial.md %})
+This workflow is also available at [proteomics.usegalaxy.eu](https://proteomics.usegalaxy.eu/)
+
+This workflow was developed by the Galaxy-P team at the University of Minnesota. For more information about Galaxy-P or our ongoing work, please visit us at galaxyp.org
+
+> ### {% icon comment %} References
```

Comment:
we have support for references in the framework, please add the citations in this box to the `tutorial.bib` file. They will then be automatically listed at the end of the tutorial in a references section. 

You can use [doi2bib](https://doi2bib.org) to generate the bibtex entries if you have a doi or PMCID for a reference, and for citing urls you can use this format:

```
@online{gtn-website,
  author = {GTN},
  title = {Galaxy Training Materials},
  url = {https://training.galaxyproject.org},
  urldate = {2020-10-13}
}
```
(And for citing them in the text you can use `{% cite gtn-webstie %}` for above example)

Reply:
Thanks for adding the bib file entries! If you put the citations in the appropriate places in the tutorial text, you don't have to make this box at all, a references sections willl automatically show up at the end.

And if some of these things aren't mentioned in tutorial yet but you still want to cite them, I would make a little section at the end with "further reading" orso, and just describe what these resources are. E.g. "To learn more about X, we recommend this video describing .. {% cite Metaproteomics_video %}" etc.